libraries/helpers.rb

```diff
@@ -0,0 +1,38 @@
+module Chocolatey
+  module Helpers
+    # Get the ChocolateyInstall directory from the environment.
+    def chocolatey_install
```

Comment:
The previous behavior would get the machine value but fall back to the user if it exists and the machine was missing in order not to break older chocolatey clients.